entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+gem install csv
```

Comment:
Nitpick: in the same vein as my other comment, we could move these `gem install`s to the dockerfile (just after installing ruby), so they get cached.

Reply:
can't immediately work out why i get an error with the json gem when i move this to the dockerfile. gonna leave as is for now and add it to my tech debt list 😕